code/modules/mob/living/living_emote.dm

```diff
@@ -71,6 +71,7 @@
 	cooldown = 10 SECONDS
 	volume = 40
 	unintentional_stat_allowed = DEAD
+	muzzle_ignore = TRUE
```

Comment:
Could you add a comment here explaining why this is set to `TRUE` even though later you do a `is_muzzled` check? In the scope of this PR it is clear enough. But 1 year later somebody will have no clue why somebody did this.

Reply:
Done ^^